eng/BuildConfiguration/build-configuration.json

```diff
@@ -0,0 +1,11 @@
+{
+   "RetryCountLimit":1,
+   "RetryByAnyError":true,
```

Comment:
This is when you want to retry when any error happens on any stage/job, I think in this case as you want that only happens in build stage so you cant set this to false :) 
or if you want that in get retry on any case you can let this on true and remove all the "RetryByPipeline" section